manifests/site.pp

```diff
@@ -70,28 +70,23 @@
 
 node 'azure.ci.jenkins.io' {
   sshkeyman::hostkey { ['azure.ci.jenkins.io']: }
-  include role::jenkins::master
-}
```

Comment:
Why do these lines have been removed? Typo?

Reply:
not a typo, but a cleanup that Ithought it was on another commit: lemme fix (I prefer separating PRs here)